scripts/pybullet_backend.py

```diff
@@ -119,10 +126,19 @@ def main():
 
         camera_data = tricamera.MultiProcessData("tricamera", True, 10)
         camera_driver = tricamera.PyBulletTriCameraDriver()
-        camera_backend = tricamera.Backend(camera_driver, camera_data)
+        camera_backend = tricamera.Backend(camera_driver, camera_data)  # noqa
 
     backend.wait_until_terminated()
 
+    if args.logfile:
+        if args.max_number_of_actions:
+            end_index = args.max_number_of_actions
+        else:
+            end_index = -1
+        logger.write_current_buffer(
+            args.logfile, start_index=0, end_index=end_index
```

Comment:
might be a stupid question: but how is the `end_index` being specified for the logger?